includes/User.php

```diff
@@ -3229,8 +3229,8 @@ public function saveSettings() {
 
 		Hooks::run( 'BeforeUserSaveSettings', array( $this ) );
 
-		$dbw = wfGetDB( DB_MASTER );
-		$dbw->update( 'user',
+		$dbw = wfGetDB( DB_MASTER, [], $wgExternalSharedDB );
+		$dbw->update( '`user`',
```

Comment:
`ExternalUser_Wikia::updateUser` does the same. Should we start removing pieces of functionality from `ExternalUser_Wikia`?

Reply:
Yes, there is no point in having two methods doing the same thing.  In addition, our expectation is that we will remove `ExternalUser_Wikia` completely or leave a small stub class.

Reply:
Should we do this in this PR or it will separate task?

Reply:
@macbre @mixth-sense removed it, tested on SUS2 sandbox.

Reply:
Thanks! One more thing - can you double-check that making an update to `wikicties.user` (for instance saving user preferences) invalidates per-cluster copies of `user` table? These are still used and we should keep the up-to-date before our changes are fully completed.